src/scope.js

```diff
@@ -400,19 +422,19 @@ class Scope {
     }
 
     __isClosed() {
-        return this.__left === null;
+        return this.__left === null || this.__left === undefined;
```

Comment:
This is a code change, I have no idea if it is significant, but thought I should point it out as it is not necessary for the PR